libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -102,12 +110,15 @@
         [style.maxHeight]="!mobile ? maxHeight : 'auto'"
         (focusEscapeList)="handleListFocusEscape($event)"
         [style.minWidth]="'100%'"
+        role="listbox"
+        aria-roledescription="Suggestion Found"
     >
         <li
             *ngFor="let value of displayedValues"
             fd-list-item
             (click)="dialogApprove()"
             [selected]="selected.indexOf(value) !== -1"
+            role="option"
         >
             <fd-checkbox
                 [compact]="compact"
```

Comment:
```
<a fd-link tabindex="0">Show All ({{ dropdownValues.length }})</a>
```

I cannot highlight line `138` in this file where you added `i18n` then removed it, but that string isn't translatable anymore, please add new `@Input()` using which customers can provide translations for the string.